Added Prefix Change

NaathySz opened the pull request:
I see a lot of peoples asking where and how they change prefix.
Most beginners who dont know compile.
Also, have this option is really better, so why not? I've added it.

splewis commented:
This change would make much more sense as a new cvar (e.g. `sm_retakes_message_prefix`) instead of changing translation files. The translation files are not meant to be customized by users, as they are expected to be overridden every plugin update.

NaathySz commented:
> This change would make much more sense as a new cvar (e.g. `sm_retakes_message_prefix`) instead of changing translation files. The translation files are not meant to be customized by users, as they are expected to be overridden every plugin update.

Reverted changes in translation and added cvar.
